sources/packages/api/src/route-controllers/note/note.aest.controller.ts

```diff
@@ -15,13 +15,16 @@ import { IUserToken } from "../../auth/userToken.interface";
 import { transformToNoteDTO, transformToNoteEntity } from "../../utilities";
 import { ApiTags } from "@nestjs/swagger";
 import { Role } from "../../auth/roles.enum";
+import { ClientTypeBaseRoute } from "../../types";
 /**
  * Controller for Notes.
  * This consists of all Rest APIs for notes.
  */
+@AllowAuthorizedParty(AuthorizedParties.aest)
+@Groups(UserGroups.AESTUser)
 @Controller("notes")
-@ApiTags("notes")
-export class NotesController extends BaseController {
+@ApiTags(`${ClientTypeBaseRoute.AEST}-notes`)
```

Comment:
I know it was already there. but @guru-aot @andrewsignori-aot @dheepak-aot @henryoforeh-aot  shouldn't @Controller, @ApiTags, and the controller name singular. did we have an agreement on that?

Reply:
I guess we had. Is it the best, not sure, but we did talk about singular for the controllers.

Reply:
I think yes we are following the singular convention.